### PR TITLE
Add configuration option to enable email notifications for extended lockout periods resulting from continuous login failures

### PR DESCRIPTION
**Pull request by DilshanSenarath**
### Related Issues
- https://github.com/wso2/product-is/issues/19419

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Comment by DilshanSenarath** [comment]
https://github.com/wso2/identity-apps/assets/74205483/a2463d86-ac7b-4f0e-9647-42ca571b5cad

File: apps/console/src/features/server-configurations/forms/login-attempt-security-form.tsx [hunk 1]
```diff
@@ -532,6 +557,25 @@ export const LoginAttemptSecurityConfigurationFrom: FunctionComponent<
...
+                    "loginAttemptSecurity.form.fields.notifyUserOnAccountLockIncrement.label") }
+                    listen={ (value: boolean) => setNotifyUserOnAccountLockIncrement(value) }
+                    checked={ notifyUserOnAccountLockIncrement }
+                    required={ false }
+                    readOnly={ readOnly }
+                    disabled={ !isConnectorEnabled }
+                    width={ 10 }
+                    data-testid={ `${testId}-notify-user-on-account-lock-time-increase` }
+                />
+                <Hint className={ "mb-5" }>
```
Comment (Achintha444) [inline apps/console/src/features/server-configurations/forms/login-attempt-security-form.tsx:573]: refrain from using helper classes for margin's. For hint, you can use the `hint` prop in the `Field.Checkbox`

File: apps/console/src/features/server-configurations/forms/login-attempt-security-form.tsx [hunk 2]
```diff
@@ -131,9 +139,11 @@ export const LoginAttemptSecurityConfigurationFrom: FunctionComponent<
 
     const [ initialConnectorValues, setInitialConnectorValues ]
         = useState<LoginAttemptSecurityFormInitialValuesInterface>(undefined);
+    const [ manageNotificationInternally, setManageNotificationInternally ] = useState<boolean>(undefined);
```
Comment (pavinduLakshan) [inline apps/console/src/features/server-configurations/forms/login-attempt-security-form.tsx:142]: What does `manageNotificationsInternally` mean? which notification will be internally managed when this is set to true?

It feels like the variable name is too generic here. Shall we consider using a better name?

**Comment by DilshanSenarath** [comment]
Since this is a somewhat urgent fix for RC1 release, can we merge this?. I will send a follow-up PR addressing review comments.

**Review by Achintha444 — APPROVED** [review]

**Pull request merged**